src/components/exerciseResult/exerciseResultSrv.js

```diff
@@ -514,6 +514,7 @@
                                 moduleResult.exercisesStatus[exerciseTypeId][exerciseId] = exerciseStatuses[exerciseTypeId][exerciseId].status;
                             }
 
+                            moduleResult.lastUpdate = Date.now();
```

Comment:
module results object is'nt changed, only the exercise results has changed.
used to 'wake up' the listener on module results object on any update